pkg/security/cert_expiry_cache.go

```diff
@@ -1,274 +0,0 @@
-// Copyright 2023 The Cockroach Authors.
-//
-// Use of this software is governed by the CockroachDB Software License
-// included in the /LICENSE file.
-
-package security
-
-import (
-	"context"
-	math_rand "math/rand"
-	"time"
-	"unsafe"
-
-	"github.com/cockroachdb/cockroach/pkg/settings"
-	"github.com/cockroachdb/cockroach/pkg/settings/cluster"
-	"github.com/cockroachdb/cockroach/pkg/util/cache"
-	"github.com/cockroachdb/cockroach/pkg/util/log"
-	"github.com/cockroachdb/cockroach/pkg/util/metric/aggmetric"
-	"github.com/cockroachdb/cockroach/pkg/util/mon"
-	"github.com/cockroachdb/cockroach/pkg/util/stop"
-	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
-	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
-)
-
-// CacheCapacityMax is set arbitrarily high; configurable later if needed.
-const CacheCapacityMax = 65000
-
-// ClientCertExpirationCacheCapacity is the cluster setting that controls the
-// maximum number of client cert expirations in the cache.
-var ClientCertExpirationCacheCapacity = settings.RegisterIntSetting(
```

Comment:
This setting is removed, it wasn't immediately clear to me what scale of client certificates the system deals with. Even if it were 1 per connecting node, I'd assume the number of nodes talking to the database in a given 24h period does not exceed 1e5.